curator_app/src/views/dashboard.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ----------------------------------------------------------------------------
+ */
+
+'use strict'
+
+const m = require('mithril')
+
+const Dashboard = {
+  view(vnode) {
+    return [
+      m('.header.text-center.mb-4',
+        m('h4', 'Welcome To'),
+        m('h1.mb-3', 'Curator'),
+        m('h5',
+          m('em',
+            'Powered by ',
+            m('strong', 'Sawtooth Simple Supply')))),
+      m('.blurb',
+        m('p',
+          m('em', 'Sawtooth Simple Supply'),
+          ' is a simple, general purpose supply chain application built',
```

Comment:
Nitpick: "general-purpose" should have a hyphen (to help parse all those adjectives before "application".  But I shall approve this PR because it's a very minor issue.